src/uniparc/components/sub-entry/SubEntryStructureSection.tsx

```diff
@@ -0,0 +1,40 @@
+import { lazy } from 'react';
+import { Card } from 'franklin-sites';
+
+import { entrySectionToLabel } from '../../config/UniParcSubEntrySectionLabels';
+
+import { UniParcSubEntryUIModel } from '../../adapters/uniParcSubEntryConverter';
+import SubEntrySection from '../../types/subEntry';
+
+const StructureView = lazy(
+  () =>
+    import(
+      /* webpackChunkName: "structure-view" */ '../../../uniprotkb/components/protein-data-views/StructureView'
+    )
+);
+
+type Props = {
+  data: UniParcSubEntryUIModel['subEntry'];
+};
+
+// TODO: this only works for uniprotkb entries but I'm not showing these here
+export const hasStructure = (data: UniParcSubEntryUIModel['subEntry']) =>
+  data.isUniprotkbEntry && data.active && data.id;
```

Comment:
Not sure if this logic is valid always.
Some UniProtKB entries don't have structures, and we'll still want to show the structures for the UniProtKB entries that will be removed soon and also for external IDs as eventually it should be based on the sequence. The issue is that we can't query the 3D Beacons API like this yet, so I guess it'll do for now

Reply:
Added this comment to the code.